rdrf/rdrf/templatetags/get_info_text_expression.py

```diff
@@ -21,7 +23,13 @@ def get_info_text_expression(fields):
         try:
             consent_section_model = ConsentSection.objects.get(pk=consent_section_model_pk)
             if consent_section_model.information_text:
-                return _(consent_section_model.information_text)
+                information_text = Template(consent_section_model.information_text)
+                context = Context({
+                    'all_available_langs': settings.LANGUAGES,
+                    'current_language': current_lang,
```

Comment:
Consider using consistent naming for variables. Where possible, use the same naming to represent language e.g. always either use "language" or "lang", but not a mix of both.
* all_available_langs
* current_language
* current_lang

Personally, I prefer the non-abbreviated form "language".